Hackerrank3.py

```diff
@@ -0,0 +1,18 @@
+# Read a given string, change the character at a given index
+# and then print the modified string.
+# replace the character at index i with character c.
+
+
+def mutate_string(string, position, character):

```

Comment:
result = string[:position]+character+string[position+!:]

Reply:
From what i understood, the coment is about me using list instead of str?
On the "task block" of this problem on Hackerrank, both methods are explained and user is encouraged to use any of them.. so I chose list.

Reply:
it's easy one-line solution and your one looks bigger and more huge.